src/components/Svg/UploadSvg.js

```diff
@@ -31,7 +34,15 @@ const UploadSvg = ({ SvgUploaded, SvgSave, ModalSvg }) => {
       archivo.name.endsWith(EXTENSION_IMAGEN_VALIDA)
     );
   };
-
+  const handleDeleteImage = (imageId) => {
+    deleteImagesMutation(imageId)
+      .then(() => {
+        Logger.info('Imagen eliminada con éxito');
+      })
+      .catch((error) => {
+        Logger.error(error);
```

Comment:
please add a message to the logger, example: ("there is an error at delete the message: ", e)